src/main/resources/assets/js/ArchiveHelper.ts

```diff
@@ -0,0 +1,33 @@
+import {ContentPath} from 'lib-contentstudio/app/content/ContentPath';
+
+interface ItemWithPath {
+    getPath(): ContentPath;
+}
+
+export abstract class ArchiveHelper {
+
+    public static filterTopMostItems(items: ItemWithPath[]): ItemWithPath[]  {
+        const result: ItemWithPath[] = [];
+
+        items.forEach((item: ItemWithPath) => {
+            const contains: boolean = result.some((itemToDelete: ItemWithPath, index: number) => {
```

Comment:
...and `itemToDelete` to `otherItem` to make it more generic, since this method can be (and is) used for other purposes than deletion.